bb-core/src/main/java/com/cognifide/qa/bb/scope/CurrentScopeHelper.java

```diff
@@ -65,7 +65,7 @@
    * @param pageObject object to verify.
    * @return true if element represented by PageObject class is visible.
    */
-  public boolean isCurrentScopeVisible(Object pageObject) {
+  public boolean isCurrentScopeVisible(Object pageObject) { //todo this should simply use BobcatWait
```

Comment:
Could we have comment above the line instead of at the end of line?
Sonar sometimes complains..
(nice to have)